AdminGUI/scripts/controls/header-admingui.control.ts

```diff
@@ -45,13 +43,13 @@ export class HeaderAdminGuiAppComponent {
      * Log Out
      * @param 
      */
-    logOut() {
-        var isLogOut = confirm("Are you sure you want to logout?");
-        if (isLogOut) {
-            this.loginService.logout();
-            window.location.href = location.origin + (location.pathname.split('/').splice(0, location.pathname.split('/').length - 3).join('/')) + '/Resources/AspNetFormsAuth/Login.html?ReturnUrl=%2f';
+    //logOut() {
```

Comment:
Commented out code is kind of scary to readers (at least to me). Can it be completely removed?

Reply:
done